### PR TITLE
Compatibility to Julia 1.5

### LINKED ISSUE #59: Hang under Julia 1.5 and nightly.

### PR DESCRIPTION
**Pull request by KDr2**
Fix #59.

### CODE REVIEW

File: src/taskcopy.jl [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Utility function for self-copying mechanism
 
-n_copies() = n_copies(current_task())
+_current_task() = ccall((:vanilla_get_current_task, libtask), Ref{Task}, ())
```
Comment (yebai) [inline src/taskcopy.jl:3]: what's the difference between these functions: `_current_task` and the standard `current_task`?

File: deps/build.jl [hunk 2]
```diff
@@ -25,16 +25,16 @@ length(products_tmp) == 0 && (products_tmp = [products[end]])
...
 
 # Listing of files generated by BinaryBuilder:
 download_info = Dict(
-    Linux(:aarch64, libc=:glibc) => ("$bin_prefix/Libtask.v0.3.0.aarch64-linux-gnu.tar.gz", "e541c0df11d48b45e9334018c014ae0d012b75f92ffbc49fc4ab6eed35593731"),
-    Linux(:i686, libc=:glibc) => ("$bin_prefix/Libtask.v0.3.0.i686-linux-gnu.tar.gz", "7f68bd21609adf35878f9b16f56151ab5f6f609999a40291ed270c1a4d0331a0"),
-    Windows(:i686) => ("$bin_prefix/Libtask.v0.3.0.i686-w64-mingw32.tar.gz", "62950b53a892fb8c699ddbb4ccc9f19b12681f64fbf85a546345d5b0f23dc8d7"),
-    MacOS(:x86_64) => ("$bin_prefix/Libtask.v0.3.0.x86_64-apple-darwin14.tar.gz", "fff6523dde93e6dc12b96a8bf1e3c35a78d8dba7cdc95cd0e5dd13225d1972df"),
-    Linux(:x86_64, libc=:glibc) => ("$bin_prefix/Libtask.v0.3.0.x86_64-linux-gnu.tar.gz", "be4c590e8f13df31855be20f31c1c4ce98f499fc5173cd35c8e8b72b79e0dc5a"),
-    Windows(:x86_64) => ("$bin_prefix/Libtask.v0.3.0.x86_64-w64-mingw32.tar.gz", "267473eb211e5060b98cede7f30336d2a7453542617521f27941d353dcfe42e8"),
+    Linux(:aarch64, libc=:glibc) => ("$bin_prefix/Libtask.v0.3.1.aarch64-linux-gnu.tar.gz", "ad342e90969280fcd198993947fb27a052357c278e40645574b93f13b9f49f25"),
```
Comment (yebai) [inline deps/build.jl:32]: Minor question: do we now need to update these hash numbers manually every time we make a new binary release?

File: src/taskcopy.jl (hunk 1 above)
Comment (KDr2) [reply src/taskcopy.jl:3]: You can find the difference here: https://github.com/JuliaLang/julia/pull/32812, this or change jl_current_task to a hand-optimized version and it breaks our lib somehow...

File: deps/build.jl (hunk 2 above)
Comment (KDr2) [reply deps/build.jl:32]: Yeah, every time release a new version of the binary we should update them, and these hashes are generated by a script provided by Yggdrasil.

Comment (yebai) [reply deps/build.jl:32]: Maybe add some info to README on how to update these?

Comment (KDr2) [reply deps/build.jl:32]: sure, got a power failure in my community today, will add the doc tomorrow.

Comment (KDr2) [reply deps/build.jl:32]: Docs added.

**Pull request merged**